dcc-storage-server/pom.xml

```diff
@@ -156,6 +156,21 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
           </execution>
         </executions>
       </plugin>
+      
+      <!-- Jar -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifest>
+              <mainClass>org.icgc.dcc.storage.server.ServerMain</mainClass>

```

Comment:
The spring-boot-maven-plugin manages the "main-class" and "start-class" entries anyways. So specifying those values in the maven-jar-plugin configuration is ignored. Manifest files always get the appropriate entries:
Main-Class: org.springframework.boot.loader.JarLauncher
Start-Class: org.icgc.dcc.storage.client.ClientMain